design-proposals/emts-scale-provisioning.md

```diff
@@ -0,0 +1,27 @@
+# Design Proposal: Scale provisioning of EMT-S edge node supporting OXM workflow
+
+Author(s): EIM team.
+
+Last updated: 29.04.2025
+
+## Abstract
+
+The Edge Microvisor Toolkit Standalone (EMT-S) node is designed to enable enterprise customers and developers to evaluate Intel silicon-based platforms for Edge AI use cases. In this context, Original Equipment Manufacturers (OXMs) play a critical role by preparing edge nodes in bulk for end customers at their facilities before shipping them to the deployment locations. To support the OXM workflow for Edge AI use cases, it is essential to implement a scalable provisioning solution for multiple EMT-S nodes. This document outlines the design proposal for enabling large-scale provisioning of EMT-S edge nodes to meet the requirements of the OXM workflow.
+
+### Proposal Summary
```

Comment:
We need more details on the workflow itself. This comes to my mind:
- what does boots require to perform provisioning? Do we need to extend the data model? Does it need MAC address? 
- how the interaction between Inventory and boots will look like? Do we need any additional component that coordinates the process or will it be a job of OM ?